vendor/pischem/caddy/Makefile

```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=            caddy
+PLUGIN_VERSION=         1.5.0
+PLUGIN_DEPENDS=         caddy
```

Comment:
This is „caddy-custom“ actually 😉

Reply:
The plugin name, or the dependency? Because it will pull any caddy*.pkg with this dependency, it worked for me.

Reply:
I've never seen that happening. It should do  single match. I think in your repo it's called "caddy" so it works as is? in opnsense repo it's only "caddy-custom".